docs/lob_guide/src/asciidoc/_chapters/introduction.adoc

```diff
@@ -0,0 +1,152 @@
+////
+/**
+* @@@ START COPYRIGHT @@@
+*
+* Licensed to the Apache Software Foundation (ASF) under one
+* or more contributor license agreements.  See the NOTICE file
+* distributed with this work for additional information
+* regarding copyright ownership.  The ASF licenses this file
+* to you under the Apache License, Version 2.0 (the
+* "License"); you may not use this file except in compliance
+* with the License.  You may obtain a copy of the License at
+*
+*   http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*
+* @@@ END COPYRIGHT @@@
+*/
+////
+
+[#introduction]
+= Introduction
+
+[#what is lOB]
+== What is LOB
+
+LOB (Large Object), a set of large object data types used to store large volumes of data, provides random and piece-wise access to the data. 
+
+Database support for LOB is not universal.
+
+[#lob data types]
+== LOB Data Types
+
+The following are the data types supported by Trafodion. +
+
+
+[caption="Table 1-1 "]
+.Descriptions for BLOB and CLOB
+[cols="30%,70%",options="header"]
+|===
+| *Data Type*         | *Desciprtion*
+| BLOB                | Binary Large Object. + 
+
+Holds large blocks of unstructured data in binary format. +
+
+BLOB is ideal for storing and handling unstructured data, such as images, audio, video and other multimedia objects.
+| CLOB                | Character Large Object. +
+
+Stores unusually large amounts of string data in character set format. +
+
+CLOB is ideal for storing and processing semi-structured data, such as large document or string.

```

Comment:
"... such as a large document or string" is better. Alternatively, "...such as large documents or strings"

Reply:
Thanks Dave.
I've corrected. :)